rpc_component/cli.py

```diff
@@ -425,7 +441,28 @@ def parse_args(args):
     )
 
     dl_parser = dep_subparsers.add_parser("download-requirements")
-    dl_parser.add_argument("--download-dir", default="./")
+    dl_parser.add_argument(
+        "--download-dir",
+        default="./",
+        help="The dependency component repo is cloned to this path (default=./)"
+    )
+
+    dt_parser = subparsers.add_parser("dependents")
```

Comment:
That's funny because my initial approach to this task required the `get` parser, but Hugh indicated that it wasn't necessary and suggested the following arguments instead `component --releases-dir <dir> dependency --component-name <component> get-dependants`.  I didn't take this approach because it would require bypassing sections of code that aren't used in the same way as the `dependency` option uses them.  I'm fine with using the `get` pattern, so I'll go ahead and make that change.  Actually, it'd be better if you just leave a comment showing the command line syntax you prefer.